lint/rule_enabled.go

```diff
@@ -68,7 +68,7 @@ var descriptorDisableChecks = []func(d desc.Descriptor) bool{
 func ruleIsEnabled(rule ProtoRule, d desc.Descriptor, l *dpb.SourceCodeInfo_Location,
 	aliasMap map[string]string, ignoreCommentDisables bool) bool {
 	if(rule.GetRuleType() == MustRule) {
```

Comment:
Is this bug in the original linter? we should upstream if so.

Reply:
The Must/Should thing is an addition that I made to our linter only. Should've caught that earlier when I made the change.